### PR TITLE
User input

### PR DESCRIPTION
**Pull request by EVWatson**

### CODE REVIEW

File: tests/CellGridTest.java [hunk 1]
```diff
@@ -1,40 +1,39 @@
...
+        CellGrid cellGrid = new CellGrid(6,6);
+
+        int expectedColumns = 6;
+        int actualColumns = cellGrid.getColumns();
+
+        assertEquals(expectedColumns, actualColumns);
+    }
+
+    @Test
+    public void getCellIsDeadReturnsCurrentStateOfCellAtSpecifiedCoordinates(){
```
Comment (jonathan-loe) [inline tests/CellGridTest.java:32]: is this test accurate?

File: src/Main.java [hunk 2]
```diff
@@ -2,18 +2,48 @@
...
+            System.err.println(message.getMessage());
+            return;
+        }
+        String[][] newCellGrid;
+
+
+
+        for(int turns = 0; turns < 10; turns++){
+//          print
+            newCellGrid = CellGridTranslator.getCellGridAsStringArray(cellGrid);
```
Comment (jonathan-loe) [inline src/Main.java:38]: I understand `CellGridTranslator` is responsible of translating the cell state into `x` or `o`. Do you see `CellGridTranslator` as part of `ConsolePrinter`?

File: tests/CellGridTest.java (hunk 1 above)
Comment (EVWatson) [reply tests/CellGridTest.java:32]: probably not

File: src/Main.java (hunk 2 above)
Comment (EVWatson) [reply src/Main.java:38]: there is a bit of an overlap at the moment; probably need to do some refactoring

**Comment by jonathan-loe** [comment]
you are on the right track. I like the class separation. 💯

**Pull request merged**